app/lib/audit/backend.dart

```diff
@@ -93,14 +100,34 @@ class AuditBackend {
     );
   }
 
+  @visibleForTesting
+  String nextTimestamp(DateTime last, DateTime next) {
+    final nextDayStart =
+        DateTime.utc(next.year, next.month, next.day).add(Duration(days: 1));
+    return nextDayStart.isBefore(last) && nextDayStart.isAfter(next)
```

Comment:
Can `nextDayStart.isAfter(next)` ever be false?

If not maybe add an `assert(nextDayStart.isAfter(next));` (leaving the condition too, is not wrong, since we won't have asserts in production).

Reply:
If the `next` timestamp is `00:00:00.000`, then they are equal, so that is just checking an unlikely edge case.